house/tests/house_tests.py

```diff
@@ -37,7 +37,7 @@ def generate_house(generate_country, generate_city):
 
 
 @pytest.mark.django_db
-class TestCourseModel:
+class TestHouseModel:
```

Comment:
Im pretty sure this change and most of the changes in this commit were already merged so this needs to be rebased